### PR TITLE
Valve.SteamCMD version latest

### PR DESCRIPTION
**Pull request by superusercode**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/109831&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/v/Valve/SteamCMD/latest/Valve.SteamCMD.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Valve.SteamCMD
+PackageVersion: latest
+PackageLocale: en-US
+Publisher: Valve Software
+PackageName: SteamCMD
```
Comment (stephengillie) [inline manifests/v/Valve/SteamCMD/latest/Valve.SteamCMD.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://www.valvesoftware.com/
PrivacyUrl:   https://store.steampowered.com/privacy_agreement/
PackageName: SteamCMD
PackageUrl: https://developer.valvesoftware.com/wiki/SteamCMD
```

**Comment by stephengillie** [comment]
Normally, the version `latest` is not allowed, but here it matches the Apps Settings:
image

and Control Panel:
image

**Review by stephengillie — APPROVED** [review]

**Comment by OTonGitHub** [comment]
how will running `winget upgrade --all` update this package since grabbing latest version depends on the static CDN link provided by valve? Is that the only way? A way to see and at least maintain versions for the package internally through winget would be nice.

**Pull request merged**